test/lint/lint-shell.sh

```diff
@@ -50,8 +50,8 @@ for descriptor in $(git ls-files -- 'contrib/gitian-descriptors/*.yml')
 do
     echo
     echo "$descriptor"
-    # Use #!/bin/bash as gitian-builder/bin/gbuild does to complete a script.
-    SCRIPT=$'#!/bin/bash\n'$(yq -r .script "$descriptor")
+    # Use #!/usr/bin/env bash to complete script with preferred shebang approach.
+    SCRIPT=$'#!/usr/bin/env bash\n'$(yq -r .script "$descriptor")
```

Comment:
Please do not change these lines as it is intended to mimic the way `gitian-builder/bin/gbuild` does: https://github.com/bitcoin/bitcoin/pull/17361#discussion_r342189958

Reply:
ah gotcha! Thank you for the clarification - will remove this change